src/index.js

```diff
@@ -12,5 +15,7 @@ import './assets/fonts/fontello/css/fontello.css';
 
 angular
   .module('app', [adbViewModule, 'ui.router', 'pascalprecht.translate'])
+  .constant('_', _)
```

Comment:
Did you go through the app and search for places where this was assumed to be globally availalbe and switch it with this DI'ed version?